knative-eventing-broker-filter/rockcraft.yaml

```diff
@@ -0,0 +1,70 @@
+# Based on ko image: https://github.com/knative/eventing/tree/knative-v1.16.1/cmd/broker/filter
+name: knative-eventing-broker-filter
+summary: Knative Eventing Broker Filter
+description: "Knative Eventing Broker Filter"
+version: "1.16.1"
+license: Apache-2.0
+base: ubuntu@22.04
+run-user: _daemon_
+
+platforms:
+    amd64:
+
+environment:
+  # Required due to the go codebase relying on the OS Env being set
+  # See https://github.com/knative/operator/blob/knative-v1.16.0/pkg/reconciler/common/releases.go#L36
```

Comment:
this comment does not apply for this rock (it applies to operator rock), although we should still include the env to be consistent with the upstream image.
I suggest to change the comment saying the env is set in upstream image which can be viewed with `docker inspect`